ui/src/components/connections/ConnectionDetails.tsx

```diff
@@ -0,0 +1,166 @@
+      ? connection.data.credentials
+      : []; */
+
+  return (
+    <SiderLayoutContent
+      description="View connection information, credential attribute data. Revoke and delete issued credentials."
+      showBackButton
+      showDivider
+      title="Connection details"
+    >
+      <Space direction="vertical" size="large">
+        <Card>
+          {(() => {
+            switch (connection.status) {
+              case "pending":
+              case "loading": {
+                return <LoadingResult />;
+              }
+              case "failed": {
+                return <ErrorResult error={connection.error.message} />;
+              }
+              case "successful":
+              case "reloading": {
+                return (
+                  <Space direction="vertical" size="middle">
+                    <Row align="middle" justify="space-between">
+                      <Card.Meta title="Connection" />
+                      <Button danger icon={<IconTrash />} type="text">
+                        Delete connection
+                      </Button>
+                    </Row>
+                    <Card className="background-grey">
+                      <Row justify="space-between">
+                        <Typography.Text type="secondary">{IDENTIFIER}</Typography.Text>
+                        <Typography.Text
+                          copyable={{
+                            icon: [<IconCopy key={0} />, <IconCheckMark key={1} />],
+                            text: connection.data.userID,
+                          }}
+                        >
+                          {obfuscateDID(connection.data.userID)}
+                        </Typography.Text>
+                      </Row>
+                      <Row justify="space-between">
+                        <Typography.Text type="secondary">Creation date</Typography.Text>
+                        <Typography.Text>
+                          {formatDate(connection.data.createdAt, true)}
+                        </Typography.Text>
+                      </Row>
+                    </Card>
+                  </Space>
+                );
+              }
+            }
+          })()}
+        </Card>
+        {isAsyncTaskDataAvailable(connection) && (
+          <TableCard
+            defaultContents={
+              <>
+                <Avatar className="avatar-color-cyan" icon={<IconCreditCardRefresh />} size={48} />
+
+                <Typography.Text strong>No credentials issued</Typography.Text>
+
+                <Typography.Text type="secondary">
+                  Credentials for this connection will be listed here.
+                </Typography.Text>
+              </>
+            }
+            isLoading={isAsyncTaskStarting(connection)}
+            onSearch={() => null}
+            query=""
+            searchPlaceholder="Search credentials, attributes..."
```

Comment:
`Search connections, credentials...`

Reply:
Should we then [amend the design](https://www.figma.com/file/BE3tUbvtSB2EqmJsybpeFC/Polygon-ID-Platform?node-id=4749-162667&t=KoTPRUTLBtQ183Ui-0) @Figmintojas?